code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -249,7 +249,8 @@ GLOBAL_LIST_INIT(strippable_monkey_items, create_strippable_list(list(
 /obj/item/organ/brain/tumor/Remove(mob/living/carbon/C, special, no_id_transfer)
 	. = ..()
 	//Removing it deletes it
-	qdel(src)
+	if(QDELETED(src))
+		qdel(src)
```

Comment:
I don't know what you're trying to do here, but this isn't it. You can't `qdel()` something that is qdeleted.

Reply:
Did you mean `if(!QDELETED(src))`

Reply:
Why did i yeh thats what this was supposed to be

Reply:
About the why this is needed organ Destroy calls Remove